docs/source/changelog.md

```diff
@@ -1,8 +1,56 @@
-# Summary of changes in Jupyter Kernel Gateway
+# Summary of changes
 
-See `git log` for a more detailed summary.
+See `git log` and `CHANGELOG.md` for a more detailed summary of changes.
```

Comment:
The RTD version of the changelog.md will likely evolve over time to summary-changes.md which will likely have a more concise description of changes.